waiter/test/waiter/scaling_test.clj

```diff
@@ -166,7 +166,10 @@
                         {:basic-authentication {:src-router-id src-router-id} :route-params {:service-id test-service-id}}))
                     {:keys [body headers status]} (async/<!! response-chan)]
                 (is (= 200 status))
-                (is (= {"content-type" "application/json", "x-cid" correlation-id} headers))
+                (is (= {"content-type" "application/json"
```

Comment:
Could factor this out in a similar way.
(defn verify-response-headers 
  [headers expected-corrolation-id]
  (is ....))

Reply:
Introduced a new variable to hold the expected headers and left the assertions in the tests.